CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.2.0
+
+- Fix purchasing of a product not initiating the purchase in iOS (https://github.com/RevenueCat/purchases-hybrid-common/pull/146/)
+- Changed `intro_price` field in the StoreProduct map to contain the Introductory Price JSON object. `introPrice` has been removed. Removal of `intro_price_cycles`, `intro_price_period`, `intro_price_period_number_of_units`, `intro_price_period_unit`, `intro_price_string` in the iOS map. (https://github.com/RevenueCat/purchases-hybrid-common/pull/147/)
```

Comment:
I did a minor because of this, just to be safe like we did with 3.1.0

Reply:
It's technically a major since it's a breaking change. I don't really mind it since we're the only clients of this, but doing a major could be a good way to warn others updating hybrids that there are also breaking changes to look out for here